src/lib.rs

```diff
@@ -133,6 +133,9 @@ pub type ssize_t = isize;
 pub enum FILE {}
 pub enum fpos_t {} // TODO: fill this out with a struct
 
+#[cfg(not(any(dox, target_env = "msvc")))]
```

Comment:
I actually don't know what the purpose of `dox` is; I'm not sure what a shim would look like. Do you mean adding a definition of `isize::MAX` inside `doc.rs`?

As I've discovered, the Windows MSCV ABI doesn't define `SSIZE_MAX` but the Windows GNU ABI does. I suppose the correct solution might be to break out a msvc and a gnu modules, and use `cfg_if` to use them. What do you think?

PS: I can't work on any of this now until Monday.